Shared/Sources/AutoResizingMaskViewSource.swift

```diff
@@ -118,11 +116,50 @@ final class AutoResizingMaskViewSource {
     return rightFlexibleTopBottomView
   }()
 
-  private func bakeViewWithFrame(_ frame: CGRect) -> UIView {
-    let view = UIView(frame: frame)
+  private func bakeViewWithFrame(_ frame: CGRect) -> UserInterfaceStyleAwareView {
+    let view = UserInterfaceStyleAwareView(frame: frame)
     view.layer.cornerRadius = type(of: self).cornerRadius
     view.layer.borderWidth = type(of: self).borderWidth
-    view.layer.borderColor = type(of: self).borderColor
+    view.layer.borderColor = view.borderColor
     return view
   }
 }
+
+class UserInterfaceStyleAwareView: UIView {
+
+  var borderColor: CGColor {
```

Comment:
Well, `fileprivate` only matters for non top level symbols - so here it won't matter. `private` is for the current scope - which is the file in that case.